Sources/Danger/Danger.swift

```diff
@@ -54,43 +59,35 @@ final class DangerRunner {
         }
 
         logger.debug("Setting up to dump results")
-        dumpResultsAtExit(self, path: outputJSONPath)
+        outputPath = outputJSONPath
+        dumpResults()
     }
-}
-
-// MARK: - Public Functions
-
-// swiftlint:disable:next identifier_name
-public func Danger() -> DangerDSL {
-    DangerRunner.shared.dsl
-}
-
-// MARK: - Private Functions
-
-private var dumpInfo: (danger: DangerRunner, path: String)?
-
-private func dumpResultsAtExit(_ runner: DangerRunner, path: String) {
-    func dump() {
-        guard let dumpInfo = dumpInfo else { return }
-        dumpInfo.danger.logger.debug("Sending results back to Danger")
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/Danger.swift;
  Line: 65;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)